node/query-redis-labs/src/main.js

```diff
@@ -0,0 +1,21 @@
+import { throwIfMissing } from "./utils.js";
+import { getClient, getWarehouse, insertTestData } from "./redis.js";
+
+/**
+ * Global connection. Reused between executions
+ */
+let client;
+
+export default async ({ req, res, log, error }) => {
+    throwIfMissing(process.env, ['REDIS_HOST', 'REDIS_PASSWORD']);
+
+    if (!client) {
+        client = await getClient();
+    }
+
+    const hash = await insertTestData(client);
+
+    const warehouse = await getWarehouse(client, hash);
+
+    return res.json(warehouse);
```

Comment:
for all the functions can we have a GET handler and throw an error on any other requests?

Reply:
Hmm good idea. But maybe we should only accept POST? If they use GET, they then also do GET for favicon.ico, and other stuff - automatic by browser.
If we only allow POST, they need proper HTTP tool to hit it (or from Appwrite Console)

Reply:
But as far as this function is concerned we are only getting data right? So it should be fine

Reply:
We also `insertTestData`